PDSA_System/Server/Models/Statistikk.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+namespace PDSA_System.Server.Models
+{
+    public class Statistikk
+    {
+        public int ForfatterId { get; set; }
+
+        public int TeamId  { get; set; }
+
+        public string Fornavn { get; set; }
+
+        public string Etternavn { get; set; }
+
+        public string Status { get; set; }
+
+        public int Stat { get; set; }
```

Comment:
Kunne det vært mulig å få nærmere dokumentasjon på hva dette atributtet betyr? Helst burde det hete `Count` dersom det alltid vil peke mot et antall

```suggestion
        public int Count { get; set; }
```

Reply:
Takk. Årsaken til at jeg ikke bruker Count som attributt er fordi Count er et reservertnøkkel ord i SQL språket. Så frykter at Count(*) AS Count kan forårsake feil.

Reply:
Jeg gjorde et forsøk på det, og det ser ut til å fungere